build-system/tasks/helpers.js

```diff
@@ -717,14 +723,82 @@ function buildExaminer(options) {
  * @param {!Object} options
  */
 function buildWebWorker(options) {
-  const opts = Object.assign({}, options);
   return compileJs('./src/web-worker/', 'web-worker.js', './dist/', {
     toName: 'ww.max.js',
     minifiedName: 'ww.js',
     includePolyfills: true,
-    watch: opts.watch,
-    minify: opts.minify || argv.minify,
+    watch: options.watch,
+    minify: options.minify || argv.minify,
+  });
+}
+
+/**
+ * Build AMP experiments.js.
+ *
+ * @param {!Object} options
+ */
+function buildExperiments(options) {
```

Comment:
Move this back to `dist.js`, since it's only used in there.

Reply:
done.